vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet.cc

```diff
@@ -138,6 +143,7 @@ void Wallet::DisconnectWallet(
             return;
           }
           ledger_->uphold()->DisconnectWallet(true);
```

Comment:
This duplicated logic looks very fragile. We shouldn't be duplicating the identical code for every single case (speaking of which, this should also be a `switch` block).

Reply:
Filed a follow-up issue to refactor duplicated code in `bat-native-ledger`: https://github.com/brave/brave-browser/issues/16796